### PR TITLE
allow specifying an amp-lightbox-gallery component within the page

### PR DESCRIPTION
**Pull request by cathyxz**
I noticed that we recommend people to use their own `amp-lightbox-gallery` tags, but this tag hasn't yet been allowed by the validator.

### CODE REVIEW

File: extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii [hunk 1]
```diff
@@ -25,6 +25,19 @@ tags: {  # amp-lightbox-gallery
   }
   attr_lists: "common-extension-attrs"
 }
+tags: {  # <amp-lightbox-gallery>
```
Comment (cathyxz) [inline extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii]: We'll need a way to also restrict this to one instance of this component per page.

Comment (honeybadgerdontcare) [reply extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii]: To limit a tag to one usage per page then you'll want to use `unique: true`. See [validator.proto](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L538).

Comment (cathyxz) [reply extensions/amp-lightbox-gallery/validator-amp-lightbox-gallery.protoascii]: Oh awesome! Thanks a lot!

**Comment by cathyxz** [comment]
Discussed offline--We won't actually need this, since adding the extension will suffice and we won't need the actual tag. We can consider revisiting when people start filing issues re: styling lightbox.

**Pull request closed without merging**